src/main/java/stirling/software/SPDF/SPdfApplication.java

```diff
@@ -39,38 +44,43 @@ public void init() {
                     SystemCommand.runCommand(rt, "rundll32 url.dll,FileProtocolHandler " + url);
                 }
             } catch (Exception e) {
-                e.printStackTrace();
+                logger.error("Error opening browser: {}", e.getMessage());
             }
         }
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException, InterruptedException {
         SpringApplication app = new SpringApplication(SPdfApplication.class);
         app.addInitializers(new ConfigInitializer());
         if (Files.exists(Paths.get("configs/settings.yml"))) {
             app.setDefaultProperties(
                     Collections.singletonMap(
                             "spring.config.additional-location", "file:configs/settings.yml"));
         } else {
-            System.out.println(
-                    "External configuration file 'configs/settings.yml' does not exist. Using default configuration and environment configuration instead.");
+            logger.warn("External configuration file 'configs/settings.yml' does not exist. Using default configuration and environment configuration instead.");
         }
         app.run(args);
 
         try {
             Thread.sleep(1000);
         } catch (InterruptedException e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
+            throw new InterruptedException("Error");
         }
```

Comment:
What is this catch block for if it re-throws the same exception?

Reply:
Thanks for the feedback. I've handled the InterruptedException
https://github.com/Stirling-Tools/Stirling-PDF/pull/853

I've re-interrupted the current thread and logged the error message. Hope it helps